spyder/widgets/panels/manager.py

```diff
@@ -0,0 +1,308 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Copyright © <2013-2016> <Colin Duquesnoy and others, see pyqode/AUTHORS.rst>
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+This module contains the panels controller, responsible of drawing panel
+inside CodeEditor's margins
+Not all panels are using PanelsManager, but future panels should use it.
+Adapted from https://github.com/pyQode/pyqode.core/blob/master/pyqode/core/managers/panels.py
+"""
+import logging
+
+from spyder.api.manager import Manager
+from spyder.api.panel import Panel
+
+
+def _logger():
+    return logging.getLogger(__name__)
+
+
+class PanelsManager(Manager):
+    """
+    Manages the list of panels and draws them inside the margin of the
+    CodeEditor widget.
```

Comment:
Change this to

    Manage the list of panels and draw them inside the margins of
    CodeEditor widgets.